addons/TBMod_rhs/CfgWeapons.hpp

```diff
@@ -170,12 +170,14 @@ class CfgWeapons
         mass = 137.75; // 38.4
     };
 
-    class rhs_weap_M136: Launcher_Base_F // M136 HEAT (CS)
+    class rhs_weap_M136;
+    class rhs_weap_M136_CS: rhs_weap_M136 // M136 HEAT (CS)
```

Comment:
So in Zukunft immer immer machen.
Ggf dann mit TB prefix in Zukunft, da es kein offizieller mehr ist.

Reply:
Zu deinem ersten Satz: ist das ein Hinweis auf ein Fehler den ich hier gemacht habe und du mir die richtige Bearbeitung zeigst oder war das nur eine Bestätigung, dass dies richtig ist und in Zukunft so beibehalten werden soll? (ich kann nämlich keinen Fehler erkennen)
Okay, Ordnung muss sein, Prefix kommt

Reply:
Also, finde es gut, das du jetzt nichts ersetzt, sondern einen neuen machst und damit nicht zu stark die RHS internen Sachen verstellst/bearbeitest, das zum ersten.
Zum Zweiten, wenn wir Sachen neu machen, dann packen wi immer TB oder so davor, das wir ganz genau sehen, das ist was von uns wenns da Probleme gibt sind wir Schuld.

Reply:
So mit!

Jup, ist ne schlechte Angewohnheit die Prefixe nicht immer durchzuziehen, dabei machen sie einem das Leben massiv einfacher.